### PR TITLE
:construction_worker: Adapt manual deployment scripts

### LINKED ISSUE #235: Unexpected token 'I', "Internal S"... is not valid JSON
Hello, I get this error as shown below, in the console when trying to view a typebot or to perform an action:
```
Unexpected token 'I', "Internal S"... is not valid JSON
```
Very similar to issue #178 

I'm using version `2.9.1` and faced the same issue as #44 before this one arose.

Additionally, I get `https://undefined/typebot-bloggers-p7dtme8` in the share link though I have crosschecked my envs configuration and checked the issue #180 and #106

Configs as follows:
```env
# Viewer
ENCRYPTION_SECRET=abc
NEXT_PUBLIC_VIEWER_URL=https://bot.example.com
DATABASE_URL=postgresql://type:bot@localhost:5432/typebot
NEXTAUTH_URL=https://builder.example.com

SMTP_USERNAME=username
SMTP_PASSWORD=pass123
SMTP_HOST=smtp.host
SMTP_PORT=587
NEXT_PUBLIC_SMTP_FROM=Notifications <notifications@example.com>
SMTP_SECURE=false
SMTP_AUTH_DISABLED=false

S3_ACCESS_KEY=4566
S3_SECRET_KEY=ghuu
S3_BUCKET=tyre
#S3_PORT=9000
S3_ENDPOINT=fra1.digitaloceanspaces.com
S3_SSL=true
```
```env
# Builder
ADMIN_EMAIL=hello@example.com
DATABASE_URL=postgresql://type:bot@localhost:5432/typebot
ENCRYPTION_SECRET=!A%D*G-KaNdRgUkXp2s5v8y/B?E(H+Mb #256-bits secret (can be gen>
NEXTAUTH_URL=https://builder.example.com
NEXT_PUBLIC_VIEWER_URL=https://bot.example.com

SMTP_USERNAME=username
SMTP_PASSWORD=pass123
SMTP_HOST=smtp.host
SMTP_PORT=587
NEXT_PUBLIC_SMTP_FROM=Notifications <notifications@example.com>
SMTP_SECURE=false
SMTP_AUTH_DISABLED=false

S3_ACCESS_KEY=4566
S3_SECRET_KEY=ghuu
S3_BUCKET=tyre
#S3_PORT=9000
S3_ENDPOINT=fra1.digitaloceanspaces.com
S3_SSL=true
```

### PR DESCRIPTION
**Pull request by baptisteArno**
Closes #235

### CODE REVIEW

**Comment by itzomen** [comment]
@baptisteArno I think the apps/viewer build command would need a similar update as the apps/builder build command

**Comment by baptisteArno** [comment]
Indeed! Can you do it and let me know if it all works?

File: apps/builder/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "license": "AGPL-3.0-or-later",
   "scripts": {
     "dev": "cross-env ENVSH_ENV=.env.local sh ../../env.sh next dev -p 3000",
-    "build": "next build",
+    "build": "cross-env ENVSH_ENV=.env.local sh ../../env.sh next build",
```
Comment (itzomen) [inline apps/builder/package.json:7]: @baptisteArno this loads the env in the manual deployment but I tweak the command i.e change `sh` to `bash`
```json
"build": "cross-env ENVSH_ENV=.env.local bash ../../env.sh next build"
```

Additionally, I added to it to apps/viewer build command

**Review by itzomen — CHANGES_REQUESTED** [review]

File: apps/builder/package.json (hunk 1 above)
Comment (baptisteArno) [reply apps/builder/package.json:7]: Wondering why `sh` wouldn't work? What's your OS / distrib?

**Comment by itzomen** [comment]
Ubuntu 22

File: apps/builder/package.json (hunk 1 above)
Comment (itzomen) [reply apps/builder/package.json:7]: Linux / Ubuntu 22

Comment (baptisteArno) [reply apps/builder/package.json:7]: What error do you have when running the script with `sh`?

Comment (itzomen) [reply apps/builder/package.json:7]: Here is the output:
```bash
-e env.sh loaded
../../env.sh: 75: [[: not found
../../env.sh: 102: Syntax error: redirection unexpected
```

**Comment by baptisteArno** [comment]
Looking forward to your simple documentation on how to deploy the apps manually 🙏

**Comment by itzomen** [comment]
> Looking forward to your simple documentation on how to deploy the apps manually @itzomen 🙏

Alright, sounds good

Expect it by Monday 🙂

**Comment by baptisteArno** [comment]
> Alright, sounds good
> 
> Expect it by Monday 🙂

You rock! Thanks 💙

**Pull request merged**